avocado/core/runners/avocado_instrumented.py

```diff
@@ -70,11 +78,22 @@ def _run_avocado(runnable, queue):
                              }]
 
             messages.start_logging(runnable.config, queue)
+
+            if run_coverage:
+                from coverage import Coverage
+                coverage = Coverage()
+                coverage.start()
+
             instance = loader.load_test(test_factory)
             early_state = instance.get_state()
             early_state['type'] = "early_state"
             queue.put(early_state)
             instance.run_avocado()
+
+            if run_coverage and coverage is not None:
```

Comment:
I can't see how `coverage` can be `None` here.  I looked at the code `class Coverage(object):`, and could not spot that possibility.

Maybe another version of this used `Coverage.current()`?

Reply:
I tried current() before this, but is not working as we would like.